docs/guides/mutations.md

```diff
@@ -197,7 +197,37 @@ Custom actions provide the cleanest way to capture specific types of mutations a
 
 - **Collection-level mutations** (`collection.update`): Simple CRUD operations on a single collection
 - **`createOptimisticAction`**: Intent-based operations, multi-collection mutations, immediately committed
-- **`createTransaction`**: Fully custom transactions, delayed commits, multi-step workflows
```

Comment:
Is `createTransaction` documented in a subsequent section? Or does this remove all references to it?